### ISSUE TITLE
Any posibility in updating?

### ISSUE THREAD

**Issue by TSAI-CHANCHANG** [sfu-gruvi-3dv/sanet_relocal_demo#4]
The environment is using torch 0.4.0 and CUDA 9.0.
However, some module is been deprecated in torch 1.5 (latest version), and it will be much harder to get torch 0.4.0 as the time goes.
Also, I install CUDA 9.0 to reproduce the result but it report `CUDA kernel failed : CUDA driver version is insufficient for CUDA runtime version`, but I have installed the driver 440.64. It seems CUDA 9.0 may not work well on my RTX GPU.
So, I just wonder if there are any possibility in updating platform. (if not, I still thank you for your great work, I just hope this could happen since I have deal with environment for nearly 1 day)

**Comment by corsy** [comment]
Yes, the pointnet module has to update the current version, I tried that but I found the accuracy drop drastically, I'm trying retrain the model.
